xbmc/cores/DllLoader/exports/util/EmuFileWrapper.cpp

```diff
@@ -73,17 +77,11 @@ void CEmuFileWrapper::CleanUp()
         delete m_files[i].file_lock;
         m_files[i].file_lock = nullptr;
       }
-#if !defined(TARGET_WINDOWS)
       //Don't memset on Windows as it overwrites our pointer
```

Comment:
See what it does? It was not memset before, cause of the PlaceHolder ptr  should not be overwritten. No the PlaceHolder is deleted prior the memsetting. That's the idea here.